Add dynamic public prefixes through constitution (#188)
microsoft/LSKV: CMakeLists.txt

add_subdirectory(proto)

option(
  PUBLIC_LEASES

>   "If enabled, all keys and values are recorded in plaintext in the ledger (insecure!)"

Reviewer: ```suggestion
  "If enabled, leases are recorded in plaintext in the ledger"
```
